src/wallet3/cli-wallet/pyproject.toml

```diff
@@ -0,0 +1,30 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = 'oxen_wallet_cli'
+authors = [
+    {name = "Your Name", email = "you@yourdomain.com"},
+]
+description = "CLI wallet for Oxen"
+readme = "README.md"
+requires-python = ">=3.8"
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "License :: OSI Approved :: MIT License",
```

Comment:
This should be `BSD-3-Clause` instead of `MIT License`.

Unless you want to go stronger; I'd be fine with `GPL-3.0-or-later`, too.  (I kind of want to relicense all of oxen-core that way).